docs/install/operator/configuring-eventing-cr.md

```diff
@@ -8,12 +8,38 @@ aliases:
 
 The Knative Eventing operator can be configured with these options:
 
+- [Version Configuration](#version-configuration)
 - [Eventing Configuration by ConfigMap](#eventing-configuration-by-configMap)
 - [Private repository and private secret](#private-repository-and-private-secrets)
 - [Configuring default broker class](#configuring-default-broker-class)
 
 __NOTE:__ Kubernetes spec level policies cannot be configured using the Knative operators.
 
+## Version Configuration
+
+As of 0.16.0, Knative users are able to install Knative Eventing of a specific version via the field `spec.version`. For
+example, if you want to install Knative Eventing 0.16.0, you can apply the following `KnativeEventing` custom resource:
+
+```
+apiVersion: operator.knative.dev/v1alpha1
+kind: KnativeEventing
+metadata:
+  name: knative-eventing
+  namespace: knative-eventing
+spec:
+  version: 0.16.0
+```
+
+If the field `spec.version` is not specified, Knative Operator will install Knative Eventing of the latest version available
+in the operator. If users specify an invalid or unavailable version, Knative Operator will do nothing. Knative Operator
+always includes 3 minor releases. For example, if the version of Knative operator is 0.16.x, the earliest version of
+Knative Eventing is 0.14.0, as available in this operator.
```

Comment:
Do we want to suggest [`0.14.2`](https://github.com/knative/eventing/releases/tag/v0.14.2), as that is the latest patch version of 0.14?

(Do we support automatic "latest patch" semver, or only exact match?)

Reply:
done.